next.config.js

```diff
@@ -0,0 +1,22 @@
+const keysTransformer = require('ts-transformer-keys/transformer').default
+
+module.exports = {
+  future: {
+    webpack5: true,
+  },
+  webpack: (config, options) => {
+    config.module.rules.push({
+      test: /\.ts$/,
+      loader: 'ts-loader',
```

Comment:
Adding this custom webpback config is required to use the `ts-transformer-keys` package